docs/advanced/classes.rst

```diff
@@ -1232,3 +1232,17 @@ appropriate derived-class pointer (e.g. using
     more complete example, including a demonstration of how to provide
     automatic downcasting for an entire class hierarchy without
     writing one get() function for each class.
+
+Accessing the type object
+=========================
+
+You can get the type object from a C++ class that has already been registered using:
+
+.. code-block:: python
+
+    py::type T_py = py::type::of<T>();
+
+You can directly use ``py::type::of(ob)`` to get the type object from any python
+object, just like ``type(ob)`` in Python.
```

Comment:
nit Is it worth adding `.. note::` here that other types like `py::type::of<int>` does not work, with a brief xref to this section?
https://pybind11.readthedocs.io/en/stable/advanced/cast/index.html

Reply:
Added, please check.

Reply:
Thanks!